sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/models/GeoPoint.java

```diff
@@ -63,6 +70,7 @@ public static GeoPoint create(double latitude, double longitude, CoordinateSyste
      *
      * @return true if valid, false if invalid
      */
+    @JsonIgnore
     public boolean isValid() {
         return coordinates != null && coordinates.size() == 2
             && coordinates.get(0) >= -180.0 && coordinates.get(0) <= 180.0
```

Comment:
`isValid` should also check that `coordinates.get(0)` and `coordinates.get(1)` are not null?